odata-protocol/11 Data Service Requests.md

```diff
@@ -386,11 +386,10 @@ The value of the `$expand` query option is a comma-separated list of
 navigation property names, stream property names, or `$value` indicating
 the stream content of a media-entity.
 
-For navigation properties, the navigation property name is optionally
```

Comment:
The text in this paragraph does not match with the new paragraph you added after this one. I think you wanted to remove the paragraph above this one instead. Kindly check.

Reply:
Thanks, I also wanted to remove the paragraph above, done now.